setup.py

```diff
@@ -37,12 +37,13 @@
     extras_require={
         'develop': [
             'pytest',
-            'bump2version'
+            'bump2version',
+            'coverage',
         ],
         'export': [
             'pandas',
             'mapbox-earcut',
-            'triangle'
+            'triangle2'
```

Comment:
@GinaStavropoulou do we want triangle2 though? This is macOS specific, others have triangle